Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -107,14 +112,237 @@ public void TearDown()
                 }
             }
             
-            // Delete temp files
-            File.Delete(_inputFilePath);
-            File.Delete(_outputFilePath);
+            // Delete temp files if necessary
+            if (_filesToDelete == null)
```

Comment:
Why not write it like
```
if (_filesToDelete != null) {
   foreach ....
}

Reply:
It is used to reduce nesting as the Rider assistant advised.